specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServers/ServerBackupV2.tsp

```diff
@@ -0,0 +1,97 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./Server.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.DBforMySQL;
+/**
+ * Server backup properties
+ */
+@parentResource(Server)
+model ServerBackupV2
+  is Azure.ResourceManager.ProxyResource<ServerBackupPropertiesV2> {
+  ...ResourceNameParameter<
+    Resource = ServerBackupV2,
+    KeyName = "backupName",
+    SegmentName = "backupsV2",
+    NamePattern = "^[-\\w\\._]+$"
+  >;
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+interface ServerBackupV2sOps
+  extends Azure.ResourceManager.Legacy.LegacyOperations<
+      {
+        ...ApiVersionParameter,
+        ...SubscriptionIdParameter,
+        ...ResourceGroupParameter,
+        ...Azure.ResourceManager.Legacy.Provider,
+      },
+      {
+        /**
+         * The name of the server.
+         */
+        @path
+        @segment("flexibleServers")
+        @pattern("^[a-z0-9][-a-z0-9]*(?<!-)$")
+        @minLength(1)
+        @maxLength(63)
+        serverName: string,
+
+        /**
+         * The name of the backup.
+         */
+        @path
+        @segment("backupsV2")
+        @pattern("^[-\\w\\._]+$")
+        backupName: string,
+      }
+    > {}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/no-resource-delete-operation" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@armResourceOperations
+interface ServerBackupV2s {
+  /**
+   * Get backup for a given server.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @operationId("LongRunningBackups_Get")
+  get is ArmResourceRead<ServerBackupV2>;
+
+  /**
+   * Create backup for a given server with specified backup name.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-put-operation-response-codes" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  #suppress "@azure-tools/typespec-azure-core/invalid-final-state" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @Azure.Core.useFinalStateVia("azure-async-operation")
+  @operationId("LongRunningBackup_Create")
+  create is ServerBackupV2sOps.CreateOrUpdateAsync<
```

Comment:
use legacy for this operation to fix https://github.com/Azure/azure-rest-api-specs/pull/35895#discussion_r2207024182